libs/gltfio/src/UbershaderLoader.cpp

```diff
@@ -105,6 +119,7 @@ Material* UbershaderLoader::getMaterial(const MaterialKey& config) const {
     }
     switch (matindex) {
         case MATINDEX(LIT, AlphaMode::OPAQUE): mMaterials[matindex] = CREATE_MATERIAL(LIT_OPAQUE); break;
+        #if !GLTFIO_LITE
         case MATINDEX(LIT, AlphaMode::MASK): mMaterials[matindex] = CREATE_MATERIAL(LIT_MASKED); break;
         case MATINDEX(LIT, AlphaMode::BLEND): mMaterials[matindex] = CREATE_MATERIAL(LIT_FADE); break;
```

Comment:
Shouldn't we at least keep blend? Without it transparents won't work and they're fairly common in glTF files. I think it's ok to ditch masked

Reply:
Could this be a compile option?

Reply:
I think these are both good ideas.  Currently the lite flag is controlled via a single CMake-supplied #define called `GLTFIO_LITE`, but we can split this up into multiple #defs to make the inclusion set configurable.  The default inclusion set should probably be OPAQUE + BLEND, as Romain suggested.

On a side note I think the CL can be simplified a little, there is no need for 2 separate resource headers (lite and non-lite), they can be combined.

Reply:
Added BLEND mode which increases the size to 538 KB, which still seems fairly reasonable.